src/kvstore/RocksEngineConfig.cpp

```diff
@@ -196,6 +199,10 @@ rocksdb::Status initRocksdbOptions(rocksdb::Options &baseOpts) {
     }
     baseOpts.table_factory.reset(NewBlockBasedTableFactory(bbtOpts));
     baseOpts.create_if_missing = true;
+    if (FLAGS_enable_prefix_bloom_filter) {
+        baseOpts.prefix_extractor.reset(rocksdb::NewFixedPrefixTransform(
```

Comment:
I have tried this way, the problem we need to fix is how to deal with the system keys inside rocksdb.  The format is not with "partId + vertexId" as its prefix.